huaweicloud/services/cce/data_source_huaweicloud_cce_nodes.go

```diff
@@ -44,6 +45,10 @@ func DataSourceNodes() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"ignore_details": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Should we add a validation for this param as it's defined by the provider?

Reply:
sure